config/index.ts

```diff
@@ -1,268 +1,406 @@
-import * as config_DEPRECATED from './config_DEPRECATED';
```

Comment:
This file is more or less net new, but because it has the same name as the old one, GH is showing a diff.